drivers/soundwire/bus_type.c

```diff
@@ -110,6 +110,9 @@ static int sdw_drv_probe(struct device *dev)
 	slave->bus->clk_stop_timeout = max_t(u32, slave->bus->clk_stop_timeout,
 					     slave->prop.clk_stop_timeout);
 
+	slave->probe_complete = true;
```

Comment:
Should we set ``slave->probe_complete = false;`` in sdw_drv_remove()?

Reply:
yes